packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -313,9 +334,11 @@ class MessageList extends Widget<Properties> {
       this._createDayHeader(timestamp);
     }
 
-    this._createMessageGroupComponent([message], author?.id);
+    this._createMessageGroupComponent([message], author?.id, true);
 
     this._scrollContentToLastMessage();
+
+    lastMessageGroup = this._messageGroups?.at(-1);
```

Comment:
[possibly excess code]
I cannot figure out what it's necessary for

Reply:
yes, it's a artifact from the previous solution, ty